scss/_patterns_navigation.scss

```diff
@@ -651,6 +651,174 @@ $spv-navigation-logo-bottom-position: 0.125rem; // 2px when 1rem is 16px
     }
   }
 
+  @keyframes fade-in {
+    from {
+      opacity: 0;
+    }
+    to {
+      opacity: 0.5;
+    }
+  }
+
+  .is-sliding {
```

Comment:
Why is this just `.is-sliding`? Is there a specific element it should be added to?
Is it `.p-navigation`? I guess it would be good to explicitly list it to make it more understandable (`.p-navigation.is-sliding`).

This makes me wonder if it should be `.p-navigation--sliding` (that would be more valid BEM name I think…)